packages/integration-karma/test/shadow-dom/event-in-shadow-tree/event-propagation.spec.js

```diff
@@ -301,6 +301,90 @@ describe('event propagation in nested shadow tree', () => {
     });
 });
 
+describe('non-composed event propagation in nested shadow tree', () => {
```

Comment:
All the tests here are for non-composed and bubbling events. 
```suggestion
describe('non-composed and bubbling event propagation in nested shadow tree', () => {
```